README.md

```diff
@@ -112,36 +112,40 @@ pipenv run pytest -x -s
 ## Usage
 
 GatorGrouper accepts command line arguments and then generates output in your
-terminal window. Using the defaults of storing your student identifiers in the
-file called `students.csv` and creating groups of two students means that you
-will run GatorGrouper with this command:
+terminal window. The program requires user to provide an input `.csv` file.
+This will create groups of three students means that you will run GatorGrouper
+with this command:
 
 ```shell
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py
+pipenv run python3 gatorgrouper_cli.py --file filepath
 ```
 
 In order to see all the possible commands and their descriptions, enter
 the following command:
 
 ```shell
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py -h
+pipenv run python3 gatorgrouper_cli.py -h
 ```
 
-### Specify File Containing List of Students
+### Details with Importing Input File
 
-If your `.csv` file is not named `students.csv`, use the `--students-file` flag
-to let GatorGrouper know the file you are using.
+Since input file is required, use the `--file` flag to let GatorGrouper know
+the path of the file you are using, or the file name if it is in the root of
+GatorGrouper repository. Please make sure the input file has more than six
+students. Otherwise you should change both group size and number of groups
+to a value larger than 1 and smaller than half of the total number of students.
 
 ```shell
-pipenv run python3 gatorgrouper/utils/gatorgrouper.py --students-file file_name.csv
+pipenv run python3 gatorgrouper_cli.py --file filepath
+pipenv run python3 gatorgrouper_cli.py --file filename.csv
```

Comment:
Why are there two commands here? Should they show the usage of setting the number of groups or size of groups?